code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -690,10 +690,16 @@
 							M = A
 							break
 				if (M)
-					message = "\red <B>[src]</B> slaps [M] across the face. Ouch!"
-					playsound(src.loc, 'sound/effects/snap.ogg', 50, 1)
+					var/mob/living/carbon/human/H = M
+					if(ishuman(H) && H.head && (H.head.flags & HEADCOVERSMOUTH))
+						message = "<span class='danger'>[src] tries to slap [H] across the face, but hits their [H.head] instead! Ouch!"

```

Comment:
</ span> is missing from the end of the message. Omit the space inside of it though, that is included here so it'll show up on Github.